README.md

```diff
@@ -142,7 +142,7 @@ Background reading: [Postgres CDC Source connector documentation](https://docs.c
 9. Expand the "Show advanced configurations" dropdown and enter the following:
    - JSON output decimal format: `NUMERIC`
    - Plugin name: `pgoutput`
-   - Tables included: `public.transaction,public.debezium_heartbeat`
+   - Tables included: `public.mc_payments, public.mc_status ,public.debezium_heartbeat`
```

Comment:
should be `public.mc_payments, public.mc_status, public.debezium_heartbeat`

Reply:
corrected in commit https://github.com/bb-mvp/kafka-pipeline/pull/36/commits/d90c8054ae1e5ab9b06b36eb2ed92ceabd4fd6ad